### ISSUE TITLE
Cannot create property '_rendered' on string 'Toast body content'

### ISSUE THREAD

**Issue by laurentfrenette** [bootstrap-vue/bootstrap-vue#3174]
### Describe the bug

I was just trying out the new toast feature and I ran into a bug. I'm not so good with Vue so it may be an issue on my end. I am just trying pop a toast on button click and I get this error message:
![image](https://user-images.githubusercontent.com/45023868/56629303-e5acca00-661a-11e9-989e-00ada3639eb2.png)

makeToast(variant = null) {
			this.$bvToast.toast('Toast body content', {
				variant: variant,
			});
		},

### Steps to reproduce the bug

1. Simply bind the makeToast method to a @click of b-button

2. Click on the button, the error message should appear

### Expected behavior

I'm expecting a toast to show up.

### Versions

**Libraries:**

 "axios": "^0.18",
        "babel-plugin-transform-object-rest-spread": "^6.26.0",
        "bootstrap": "^4.3.1",
        "cross-env": "^5.2.0",
        "eslint": "^5.16.0",
        "eslint-plugin-vue": "^5.2.2",
        "jquery": "^3.3.1",
        "laravel-mix": "^2.0",
        "popper.js": "^1.14.4",
        "vue": "^2.5.17",
        "vue-router": "^3.0.1",
        "vuejs-noty": "^0.1.3",
        "vuex": "^3.0.1"

**Environment:**

- Device: PC
- OS: Windows 10
- Browser: Chrome
- Version: 73.0.3683

### Additional context

I'm using a simple architecture, where I have an index component that renders a header, sidebar and content components.

**Comment by tmorehouse** [comment]
Are you using a build system (such as webpack or rollup)?  If so you might need to set up an alias to make sure your app and BootstrapVue are using the same Vue module/instance:

https://bootstrap-vue.js.org/docs#using-module-bundlers (see the note at the bottom of the section)

**Comment by laurentfrenette** [comment]
I am using Webpack with Laravel Mix. I tried setting up an alias for Vue in my webpack config yet couldn't fix the error. The weird thing is that I'm using a lot of other bootstrap-vue Component in my project and this one is the only one that generates this error.

What kind of additional details should I provide ? Sorry it's the first time I'm starting an issue on Github.

Thanks!

**Comment by tmorehouse** [comment]
Maybe if you could provide a sample of your main app code (where you import bootstrap-vue), and your code with calls to `this.$bvToast.toast()`.

Are you using Nuxt.js?

**Comment by laurentfrenette** [comment]
No i'm not using Nuxt.

Here's my App.js (that's from vue-SPA

`import $ from 'jquery';
import Vue from 'vue';
import VueNoty from 'vuejs-noty';
import axios from 'axios';
import VueSidebarMenu from 'vue-sidebar-menu';
import VueSweetalert2 from 'vue-sweetalert2';

import BootstrapVue from 'bootstrap-vue';
import 'bootstrap-vue/dist/bootstrap-vue.css';
import 'vue-sidebar-menu/dist/vue-sidebar-menu.css';

Vue.use(VueSidebarMenu);
Vue.use(BootstrapVue);

const options = {
	confirmButtonColor: '#41b882',
	cancelButtonColor: '#ff7674',
};
Vue.use(VueSweetalert2, options);

window.$ = window.jQuery = $;
window.axios = axios;
// eslint-disable-next-line no-undef
require('bootstrap');
Vue.use(VueNoty, {
	progressBar: false,
	layout: 'bottomRight',
	theme: 'bootstrap-v4',
	timeout: 3000,
});

import router from './router';
import store from './store/index';
import App from './components/App.vue';
import jwtToken from './helpers/jwt-token';

axios.interceptors.request.use(config => {
	config.headers['X-CSRF-TOKEN'] = window.Laravel.csrfToken;
	config.headers['X-Requested-With'] = 'XMLHttpRequest';

	if (jwtToken.getToken()) {
		config.headers['Authorization'] = 'Bearer ' + jwtToken.getToken();
	}

	return config;
}, error => {
	return Promise.reject(error);
});

axios.interceptors.response.use(response => {
	return response;
}, error => {
	const errorResponseData = error.response.data;

	const errors = ['token_invalid', 'token_expired', 'token_not_provided'];

	if (errorResponseData.error && errors.includes(errorResponseData.error)) {
		store.dispatch('unsetAuthUser')
			.then(() => {
				jwtToken.removeToken();
				router.push({name: 'login'});
			});
	}

	return Promise.reject(error);
});

Vue.component('app', App);


// eslint-disable-next-line no-unused-vars
const app = new Vue({
	router,
	store,
}).$mount('#app');
`
Here's the method :

`methods: {
		makeToast(variant = null) {
			this.$bvToast.toast('Toast body content', {
				variant: variant,
			});
		},
},`

And here's the Component markup : 
`<template>
	<div class="container">
		<b-button style="margin: 10%;" :variant="buttonVariant" @click="makeToast('primary')">
			{{ buttonText }}
		</b-button>
		<b-table :items="items" :fields="fields" :sort-by.sync="sortBy" :sort-desc.sync="sortDesc" responsive small fixed />
	</div>
</template>`

**Comment by tmorehouse** [comment]
I would suggest making a test app with a bare minimum of includes (i.e. just bootstrap-vue), and test with a button and your makeToast method and see if the problem persists, and then start adding back in the other various includes to see if there is some conflict.

**Comment by tmorehouse** [comment]
You also shouldn't need the line `require('bootstrap');` and jQuery unless something else is explicitly needing them.

**Comment by laurentfrenette** [comment]
I made a minimalist project and I was able to use the toast. Now I have to find out what exactly is causing this error from my configuration. 

Thanks for your quick answers @tmorehouse !

**Comment by atilacamurca** [comment]
@laurentfrenette Did you found what was wrong? I'm having the same issue and can't figure it out.

**Comment by tmorehouse** [comment]
@atilacamurca make sure you are using Vue 2.6.x (preferably 2.6.10), as Vue has changed how slots are rendered.

**Comment by atilacamurca** [comment]
Yes, installing Vue 2.6 resolves the problem.

One workaround with Vue 2.5 is to always pass an array to toast. For example:

```js
exports default {
    methods: {
        makeToast() {
            this.$bvToast.toast(['teste'])
        }
    }
}
```

**Comment by tmorehouse** [comment]
We might be able to address this by running the message argument through `Array.concat()` before passing to the default slot.

**Comment by tmorehouse** [comment]
Will be fixed in 2.0.0-rc.20

**Comment by laurentfrenette** [comment]
Happy to see that you found a solution to this! I ended up just using the vue-bootstap alert, in my case.

Thanks
